### PR TITLE
feat(new-tool): json minify

### PR DESCRIPTION
**Pull request by tsonglew**
add new tool:  json minify

### CODE REVIEW

File: src/tools/json-minify/index.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Braces } from '@vicons/tabler';
+import { defineTool } from '../tool';
+
+export const tool = defineTool( {
+  name: 'JSON minify',
+  path: '/json-minify',
+  description: 'Minify your JSON to a string',
```
Comment (CorentinTh) [inline src/tools/json-minify/index.ts:7]:
```suggestion
  description: 'Minify and compress your JSON by removing unnecessary withe spaces.',
```

**Review by CorentinTh — CHANGES_REQUESTED** [review]
Hi @tsonglew !

Thank you for submiting this new tool, that's a great idea!
Before merging, can you please fix the linting issues, to match the rest of the app 🙏🏻 

Thanks again for your time 💪🏻

**Comment by tsonglew** [comment]
PTAL @CorentinTh

**Review by CorentinTh — APPROVED** [review]
👌🏻

**Comment by CorentinTh** [comment]
Thank you for taking the time to submit a PR 🙏🏻

**Comment by CorentinTh** [comment]
It's live : https://it-tools.tech/json-minify

**Comment by tsonglew** [comment]
Thx !🎉🎉🎉

**Pull request merged**